clang/include/clang/Sema/Sema.h

```diff
@@ -11220,6 +11220,11 @@ class Sema final {
   VarDecl *buildCoroutinePromise(SourceLocation Loc);
   void CheckCompletedCoroutineBody(FunctionDecl *FD, Stmt *&Body);
 
+  // Heuristically tells if the function is get_return_object by matching
```

Comment:
I think it's useful to mention that `get_return_object` is coming from the coroutine's `promise_type` as defined by the C++ standard. (I think it might not be obvious).

Also, could we change the name to something like `CanBeReturnObject` to clearly indicate it's a heuristic to the users.

Reply:
Both done.